app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -369,15 +379,33 @@ export default class SampleViewV2 extends React.Component {
     });
   };
 
-  persistReportOptions = () => {
+  updateHistoryAndPersistOptions = (action = "push") => {
     // save new options to local storage
     const { selectedOptions } = this.state;
+
     // remove exceptions to persistent options
-    const { taxon, ...persistentReportOptions } = selectedOptions;
+    const { taxon, ...localStatePersistedOptions } = selectedOptions;
+
+    const urlState = pick(["view", "selectedOptions"], this.state);
```

Comment:
Should pipelineVersion be included in this `pick`? It's defined in the UrlQueryParser.

Reply:
It can be as it will be needed soon.